src/components/SingleArticleCard.jsx

```diff
@@ -44,7 +54,9 @@ export default function SingleArticleCard({article, activeUserName, id, setVoteA
             <h2 className="singleHeading">{article.title}</h2>
             <div className="createdBlock">
             <Link className="linkAuthor" to={`/article?author=${article.author}`}>Created by: <span className="spanNameAuthor">{article.author} </span></Link>
-                {/* <p className="createdDate">{dateArr[0]}</p> */}
+
+                {/* <p className="createdDate">{arrayDate[0]}</p> */}
```

Comment:
It would be good if you could see everything that it's possible to sort by on the cards so it's possible to tell if it's sorting properly (i.e. comment count as well)